src/js/tech/html5.js

```diff
@@ -200,6 +200,43 @@ class Html5 extends Tech {
     });
   }
 
+  /**
+   * Attempt to force override of native audiio/video tracks.
+   *
+   * @param {Boolean} override - If set to true native audio/video will be overridden,
+   * otherwise native audio/video will potentially be used.
+   */
+  overrideNativeTracks(override) {
+    // If there is no behavioral change don't add/remove listeners
+    if (override !== (this.featuresNativeAudioTracks && this.featuresNativeVideoTracks)) {
```

Comment:
should audio and video tracks be overridden in a separate check?

Reply:
at this time, we're assuming they'll always match, we can always change it later if necessary.